include/osquery/tables.h

```diff
@@ -846,10 +871,6 @@ class TablePlugin : public Plugin {
   static void setRequestFromContext(const QueryContext& context,
                                     PluginRequest& request);
 
-  /// Helper data structure transformation methods.
-  static void setContextFromRequest(const PluginRequest& request,
```

Comment:
API Change

Reply:
I moved it because it wasn't referenced anywhere! Thanks for spotting this!